daemon/indexing/apollo/index.js

```diff
@@ -97,6 +97,7 @@ const typeDefs = gql`
     # itemsPerPage: Int!
     # totalNumberOfPages: Int!
     nodes: [Listing]
+    maxPrice: Float
```

Comment:
Couple questions:
 - In the future there could be other aggregated fields we want to query. For ex min price, num bedrooms, etc... So should this be wrapped in an object. Like ListingStats ?
 - Since this aggregated data is used to display the range on a filter, don't we need to get that information *before* making the search query, at the time we display the filter ? If so, should this be a separate query with its own graphql entrypoint. Like listingStats and it would return a ListingStats object ?

Reply:
- yeah good idea. Thinking of naming it stats so it is more general and could also apply to different index types once we implement it (people for example)?
- The search page that displays results always has a search query preceding it. So filters displayed on that page can populate their properties from search response. The initial query a user makes is from the navigation bar (general search) and at that point the page displays no filters. So there is no need to pre-issue a search to discovery server.